tests/functional/pages/surveys/household_composition/summary.page.js

```diff
@@ -0,0 +1,36 @@
+import SummaryPage from '../../summary.page'
+
+class HouseholdCompositionSummary extends SummaryPage {
+
+  clickEdit() {
+    browser.click('[data-qa="414699da-1667-44fd-8e98-7606966884db-edit"]')
+    return this
+  }
+
+  isNameDisplayed(name) {
+    return browser.element('[data-qa="414699da-1667-44fd-8e98-7606966884db-answer"]').getText() === name
+  }
+
+  areNamesDisplayed(names) {
```

Comment:
It might be nicer if this was getHouseholdNames with a contains assertion instead of us writing that logic ourself